runtests.py

```diff
@@ -234,6 +234,10 @@ def loadTestsFromModule(self, module, *args, **kw):
 loader = Loader()
 suite = unittest.TestSuite()
 for d in ['python/tests', 'tests/integration']:
+    # On Python 3 we need to pass in a Unicode string, and on Python 2 a
+    # bytestring.
+    if sys.version_info < (3, 0):
```

Comment:
Actually I'm pretty sure I can drop this entirely since we aren't using `unicode_literals` any more.